internal/controller/factory/statefulset_test.go

```diff
@@ -137,6 +144,12 @@ var _ = Describe("CreateOrUpdateStatefulSet handler", func() {
 			By("Checking the extraArgs")
 			Expect(sts.Spec.Template.Spec.Containers[0].Command).To(Equal(generateEtcdCommand()))
 
+			By("Checking the readinessGates")
+			Expect(sts.Spec.Template.Spec.ReadinessGates).To(Equal(etcdcluster.Spec.PodTemplate.Spec.ReadinessGates))
```

Comment:
```go
By("Checking the readinessGates", func() {
  Expect(sts.Spec.Template.Spec.ReadinessGates).To(Equal(etcdcluster.Spec.PodTemplate.Spec.ReadinessGates))
})
```

Reply:
Updated. But we need to refactor other asserts this way later